behaviors/magic-button.js

```diff
@@ -297,12 +298,19 @@ module.exports = function (result, args) {
     transform = args.transform || '',
     url = args.url || '',
     property = args.property || '',
-    moreMagic = args.moreMagic ? JSON.stringify(args.moreMagic) : '',
+    // object-based element attribute values require escaping double quotes
+    moreMagic = args.moreMagic ? JSON.stringify(args.moreMagic).replace(/"/g,'\"') : '',
     input = getInput(el),
-    button = dom.create(`<a class="magic-button" rv-on-click="${name}.doMagic" data-magic-currentField="${name}" data-magic-field="${field}" data-magic-component="${component}" data-magic-transform="${transform}" data-magic-url="${url}" data-magic-property="${property}" data-magic-moremagic="${moreMagic}">
+    button = dom.create(`<a class="magic-button" rv-on-click="${name}.doMagic" data-magic-currentField="${name}" data-magic-field="${field}" data-magic-component="${component}" data-magic-transform="${transform}" data-magic-url="${url}" data-magic-property="${property}">
       <img class="magic-button-inner" src="${site.get('assetPath')}/media/components/clay-kiln/magic-button.svg" alt="Magic Button">
     </a>`);
 
+  // magic that lives in an object isn't treated kindly by template strings with dom.create()
+  // instead, add object-based magic once the dom element has been created
+  if (moreMagic) {
+    button.setAttribute('data-magic-moremagic;, moreMagic);
```

Comment:
LOL. Was trying to push out that update before my son turned into a pumpkin at daycare...